inc/BAT/Availability.php

```diff
@@ -177,6 +177,28 @@ public function add_room( Room $room ) {
 
 		return $this;
 	}
+	/**
+	 * Show the price by a type.
```

Comment:
Pls re-write the code comment. I don't know "Show the price by a type" what is mean.